### ISSUE TITLE
(Accessibility) Unable to use the keyboard to tab through selected fields and remove them

### ISSUE THREAD

**Issue by mysticflute** [forcedotcom/soql-tooling#55]
**Code Builder or Desktop**:
Code Builder

**OS and version**:
Windows

**VS Code Version**:

**Browser Version (if applicable)**:
Edge

### Summary

For accessibility, the selected fields should be in the tab order and you should be able to use the keyboard to remove them.

### Steps To Reproduce:

1. Select multiple fields in the SOQL builder
2. Press the tab key

### Expected result

The selected fields should be in the tab flow, and you should be able to use the keyboard (e.g., press enter) to remove the field.

### Actual result

Focus skips over the selected fields to the Order By dropdown.

### Additional information

_Feel free to attach a screenshot_.

**SFDX CLI Version**:
